### DISCUSSION TITLE
Assignment Submission Functionality Missing / Undocumented

### DISCUSSION THREAD

**Discussion by ssyberg** [community/community#31488] [GitHub Education]
I'm trying to figure out how students are supposed to submit their assignments. I've seen some old chatter about this in the old forum but haven't found anything applicable to 2022. Anyone know what this is? 

All of these students have passing tests and feedback PR merged, but still no indication that they were "submitted"

![image](https://user-images.githubusercontent.com/1661046/187687659-c52ae85d-3b99-4283-a21b-a51a8dc7d3e8.png)
![image](https://user-images.githubusercontent.com/1661046/187687707-a3cd8f44-0b58-480f-b717-f04b7b9cffc2.png)

**Comment by nuke-web3** [comment]
More action here: https://github.com/orgs/community/discussions/17883
Please fix this, Github :pray:  - or at least tell us what it _is_.

**Comment by nuke-web3** [comment]
If the archive is still about the same, here might be some clues: 

https://github.com/education/classroom/blob/a824a057b939c0bbbbad2a9258aa700baef80ef1/app/jobs/deadline_job.rb#L1-L39
